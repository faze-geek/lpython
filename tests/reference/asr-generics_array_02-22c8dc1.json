{
    "basename": "asr-generics_array_02-22c8dc1",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/generics_array_02.py",
    "infile_hash": "a9575810ad7ee98bcc913414faf45c3fbb2e62a42e6a085e158ae280",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_array_02-22c8dc1.stderr",
    "stderr_hash": "3828a13a97a6d27cfc645a3bca067747028deb6458dd2fe63640ec96",
    "returncode": 2
}