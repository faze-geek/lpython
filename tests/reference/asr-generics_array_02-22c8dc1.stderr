semantic error: 'i32[2]' object does not support item assignment
  --> tests/../integration_tests/generics_array_02.py:28:5
   |
28 |     a_int[0] = 400
   |     ^^^^^^^^^^^^^^ 
